[2025-04-24T06:13:34.800+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-24T06:13:34.812+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: news_etl.run_t_node manual__2025-04-24T06:13:22.136506+00:00 [queued]>
[2025-04-24T06:13:34.817+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: news_etl.run_t_node manual__2025-04-24T06:13:22.136506+00:00 [queued]>
[2025-04-24T06:13:34.817+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-04-24T06:13:34.824+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): run_t_node> on 2025-04-24 06:13:22.136506+00:00
[2025-04-24T06:13:34.831+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=166) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-24T06:13:34.832+0000] {standard_task_runner.py:72} INFO - Started process 216 to run task
[2025-04-24T06:13:34.832+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'news_etl', 'run_t_node', 'manual__2025-04-24T06:13:22.136506+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/New_ETL.py', '--cfg-path', '/tmp/tmptsr6zrh9']
[2025-04-24T06:13:34.833+0000] {standard_task_runner.py:105} INFO - Job 4: Subtask run_t_node
[2025-04-24T06:13:34.864+0000] {task_command.py:467} INFO - Running <TaskInstance: news_etl.run_t_node manual__2025-04-24T06:13:22.136506+00:00 [running]> on host a38bd757ef2e
[2025-04-24T06:13:34.911+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='news_etl' AIRFLOW_CTX_TASK_ID='run_t_node' AIRFLOW_CTX_EXECUTION_DATE='2025-04-24T06:13:22.136506+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-24T06:13:22.136506+00:00'
[2025-04-24T06:13:34.912+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-24T06:13:34.922+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-04-24T06:13:34.923+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/***/plugins/data_collection/T_node.py']
[2025-04-24T06:13:34.932+0000] {subprocess.py:99} INFO - Output:
[2025-04-24T06:13:43.981+0000] {subprocess.py:106} INFO - 
[2025-04-24T06:13:43.987+0000] {subprocess.py:106} INFO - ✅ 저장 시작
[2025-04-24T06:13:43.988+0000] {subprocess.py:106} INFO - ✅ 저장 완료
[2025-04-24T06:13:44.021+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-04-24T06:13:44.036+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-24T06:13:44.037+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=news_etl, task_id=run_t_node, run_id=manual__2025-04-24T06:13:22.136506+00:00, execution_date=20250424T061322, start_date=20250424T061334, end_date=20250424T061344
[2025-04-24T06:13:44.064+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-24T06:13:44.073+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-24T06:13:44.075+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
